templates/includes/sidebar.html

```diff
@@ -1,6 +1,19 @@
 <ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
+    {% if user.is_authenticated %}
+    <li>Hi, <a href="{% url 'taxi:driver-detail' pk=user.id %}">{{ user.username }}</a>!</li>
```

Comment:
Ensure that the `driver-detail` URL correctly resolves using `user.id` as the primary key. This is important for linking to the user's profile page as specified in the task requirements.